File: Projects/Framework-Samples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/ProductManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface IProductManager : IManagerBase<Product, string>
    {
	}
	
	partial class ProductManager : ManagerBase<Product, string>, IProductManager
    {
	}
}

File: Projects/Framework-Samples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/ProductManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface IProductManager
	'	Inherits IManagerBase(Of Product, System.String)
	'End Interface

	Partial Class ProductManager
		Inherits ManagerBase(Of Product, System.String)
		Implements IProductManager
	End Class
End Namespace
